src/rpc/rawtransaction.cpp

```diff
@@ -1755,8 +1739,9 @@ UniValue converttopsbt(const JSONRPCRequest& request)
                             "2. permitsigdata           (boolean, optional, default=false) If true, any signatures in the input will be discarde and conversion.\n"
                             "                              will continue. If false, RPC will fail if any signatures are present.\n"
                             "3. iswitness               (boolean, optional) Whether the transaction hex is a serialized witness transaction.\n"
-                            "                              If iswitness is not present, heuristic tests will be used in decoding. Only has an effect if\n"
-                            "                              permitsigdata is true\n"
+                            "                              If iswitness is not present, heuristic tests will be used in decoding. If true, only witness deserializaion\n"
```

Comment:
Nit: RPC documentation is being changed in unrelated wallet RPC commit.

Reply:
Fixed